fv/containers/containers.go

```diff
@@ -380,8 +391,12 @@ func StartTwoNodeEtcdTopology() (felixes []*Container, etcd *Container, client c
 // - Configures routes between the hosts, giving each host 10.65.x.0/24, where x is the
 //   index in the returned array.  When creating workloads, use IPs from the relevant block.
 // - Configures the Tunnel IP for each host as 10.65.x.1.
-func StartNNodeEtcdTopology(n int) (felixes []*Container, etcd *Container, client client.Interface) {
+func StartNNodeEtcdTopology(n int, topoOptions ...TopologyOptions) (felixes []*Container, etcd *Container, client client.Interface) {
 	log.Infof("Starting a %d-node etcd topology.", n)
+	var opts TopologyOptions
+	if len(topoOptions) > 0 {
+		opts = topoOptions[0]
+	}
```

Comment:
Think we should take the one time hit of adding topoOptions in all existing calls to these functions, and *not* make it an optional arg as here.  As it is, I misread the ... as meaning that there could be one set of options for the first Felix, a different set for the second Felix, and so on.